doc/sphinx/addendum/micromega.rst

```diff
@@ -27,6 +27,12 @@ rationals ``Require Import Lqa`` and reals ``Require Import Lra``.
   generating a *proof cache* which makes it possible to rerun scripts
   even without `csdp`.
 
+  opt:: Simplex
```

Comment:
Use `flag`. The refman now distinguishes between non-boolean `option` and `flag`.